qmarkdowntextedit.h

```diff
@@ -120,8 +120,8 @@ class QMarkdownTextEdit : public QPlainTextEdit {
 
    Q_SIGNALS:
     void urlClicked(QString url);
-    void zoomIn();
-    void zoomOut();
+    void zoomInTriggered();
+    void zoomOutTriggered();
```

Comment:
This will break all applications that use this signals.

Reply:
...and most of the the other method changes do as well.

Reply:
Okay, then not